lib/unsplash/photo.rb

```diff
@@ -57,10 +59,18 @@ def random(categories: nil, collections: nil, featured: nil, user: nil, query: n
           h:        height,
           orientation: orientation
         }.select { |k,v| v }
-
-        photo = Unsplash::Photo.new JSON.parse(connection.get("/photos/random", params).body)
-        photo.user = Unsplash::User.new photo.user
-        photo
+        if count
+          params[:count] = count
+          photos = parse_list connection.get("/photos/random/", params).body
```

Comment:
Yeah that's true, but honestly I'm not worried about @razasayed's initial implementation. `count` maxes out at 30 anyway - this wouldn't ever be a real life performance issue.

We can leave this as-is; I've got some plans to refactor the parsing code in the nearish future anyway. Thanks for the feedback though @codenamev. :+1: